awx/main/tasks.py

```diff
@@ -882,8 +882,11 @@ def get_path_to(self, *args):
 
     def build_execution_environment_params(self, instance):
         if instance.execution_environment_id is None:
-            self.instance = instance = self.update_model(
-                instance.pk, execution_environment=instance.resolve_execution_environment())
+            from awx.main.signals import disable_activity_stream
+
+            with disable_activity_stream():
+                self.instance = instance = self.update_model(
+                    instance.pk, execution_environment=instance.resolve_execution_environment())
```

Comment:
I don't think this should ever be needed in this new system, but it shouldn't do harm